packages/input-text/src/protectedValue.js

```diff
@@ -124,19 +126,37 @@ export default WrappedComponent => {
             focused={this.state.isEditing}
             value={this.state.isEditing ? "" : this.state.value}
             onKeyDown={covered ? this.onKeyDown : onKeyDown}
+            onChange={event =>
+              this.setState({ editedValue: event.target.value })
+            }
+            onBlur={() => {
+              this.state.isEditing && this.onCancel();
+            }}
```

Comment:
this will override any onChange and onBlur properties passed to the input.

Reply:
Sure thing... It broke some tests... Fixing